plugin/src/test/java/org/jenkinsci/plugins/structs/describable/DescribableModelTest.java

```diff
@@ -873,4 +875,134 @@ public void testJavaStandardTypes() throws Exception {
                 "doubleValue1", 0.0
         ));
     }
-}
\ No newline at end of file
+
+    @Test
+    public void given_model_when_fieldRenamed_then_uselessSettersIgnored() throws Exception {
+        EvolvedClass instance;
+        Map<String,Object> expected = new HashMap<String, Object>();
+
+        instance = new EvolvedClass(false);
+        expected.clear();
+        expected.put("option", Boolean.FALSE);
```

Comment:
Well the issue with ImmutableMap.of is that you end up having to apply type hints everywhere, e.g. 

```
assertThat("Only required properties",
                DescribableModel.uninstantiate2_(instance).toMap(), Matchers<Map<String,Object>>.is(ImmutableMap.<String,Object>of("option", Boolean.FALSE)))
```